### ISSUE TITLE
Blog Post (2017-04-26): Hello World

### ISSUE THREAD

**Issue by mikeeus** [OSSDC/blog.ossdc.org#1]
http://ossdc.mikias.net/2017/04/26/hello-world/

**Comment by jatindhankhar** [comment]
@mikeeus 
You are using Wordpress but we AFAIK we discussed to stay from Wordpress and other traditional CMS. My main concern is the server side code which you are uploading in this open repo. I don't think that should be the way. 
Let's discuss this thoroughly then only pull this off
@mslavescu

**Comment by mslavescu** [comment]
@jatindhankhar you are right, we shouldn't use wordpress for serving as we prior discussed, the idea here is to use it only for editing, by OSSDC.org authors, then push the article to this repo as static content for serving through GitHub pages.

The comments section would be handled also through GitHub comments, by creating an associated issue on this repo, tagged as blog comment, for each article.

Once an author publishes an article on the (edit side of the) blog, will need to create an issue here and refer it at the end of the article to invite readers to comment.

See for example my first post:

http://ossdc.mikias.net/2017/04/29/a-new-ossdc-org-mvp-to-build-a-smart-dash-camera-using-android-phones/

This way we simplify the collaboration and make use of all Github free resources.

Once a commenter has a GitHub account it is easier to contribute to the code also.

**Comment by jatindhankhar** [comment]
@mslavescu Okay but Github pages cannot serve wordpress pages directly, also there is no need to expose server side code,database dump and php files here. 
You are also exposing sensitive information, take a look at following code 
[https://github.com/OSSDC/blog.ossdc.org/blob/master/db/database_dump.sql#L437](https://github.com/OSSDC/blog.ossdc.org/blob/master/db/database_dump.sql#L437)

```php
INSERT INTO `wp_users` VALUES (1,'mikias','$P$Bp4xyrkn/3VlAdv/QOKgfsFOeuO.cd0','mikias','itsmikias@gmail.com','','2017-04-27 04:57:52','',0,'mikias');
```

**Comment by mikeeus** [comment]
@jatindhankhar You're absolutely correct, I'm going to delete the server side code from the repo and the only thing that will be in there is statically generated code which will work on Github pages.

**Comment by mikeeus** [comment]
By the way the php files and db dump are from a one off wp site I made locally so none of the info applies to the website currently online.
